plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -354,6 +355,11 @@ public IcebergSessionProperties(
                         "Require filter on partition column",
                         icebergConfig.isQueryPartitionFilterRequired(),
                         false))
+                .add(booleanProperty(
+                        INCREMENTAL_REFRESH_ENABLED,
+                        "Enable Incremental refresh for MVs backed by Iceberg tables, when possible.",
+                        false,
```

Comment:
since it's not hidden session property, add also a feature toggle, e.g: `icebergConfig.isIncrementalRefreshEnabled`